packages/crypto-transaction-multi-payment/source/versions/1.ts

```diff
@@ -60,27 +63,34 @@ export class MultiPaymentTransaction extends Transaction {
 		return data;
 	}
 
+	public assetSize(): number {
+		const { data } = this;
+		Utils.assert.defined<Contracts.Crypto.IMultiPaymentItem[]>(data.asset?.payments);
+		const { payments } = data.asset;
+
+		return (
+			2 + // number of payments
+			payments.length * 8 + // amounts
+			payments.length * this.addressSize // recipients
+		);
+	}
+
 	public async serialize(options?: Contracts.Crypto.ISerializeOptions): Promise<ByteBuffer | undefined> {
```

Comment:
Return only ByteBuffer

```suggestion
	public async serialize(options?: Contracts.Crypto.ISerializeOptions): Promise<ByteBuffer> {
```